Authentication/SignUp/Controller/SignupControllerDelegate.swift

```diff
@@ -142,7 +142,7 @@ public protocol SignupControllerDelegate {  //swiftlint:disable:this class_deleg
     
 }
 
-extension SignupControllerDelegate {
+public extension SignupControllerDelegate {
```

Comment:
We can leave it as `public`. I asked because it was `internal` before and didn't think we're exposing anything new here, just making modifiers explicit.